src/persist/src/indexed/mod.rs

```diff
@@ -939,6 +941,33 @@ impl<L: Log, B: Blob> Indexed<L, B> {
     }
 }
 
+impl<L: Log, B: Blob> Indexed<L, B> {
+    /// Returns a [Description] of the collection identified by `id_str`.
+    fn get_description(&mut self, id_str: &str, res: PFutureHandle<Description<u64>>) {
+        res.fill((|| {
+            // TODO: Is apply_batched_cmd OK to use here?
```

Comment:
yup

Reply:
if you used `apply_batched_command` you'll need to add the response to the list of pending responses, and then by the time the responses are returned to the user, they may not be accurate anymore, if e.g. other seal / since calls were in flight at the same time as this `get_description` call. In contrast, if you use `apply_unbatched_cmd` the return value will definitely be accurate at the time of the return.

I'm not sure if this is a big deal for the intended use case, because our batching is pretty minimal rn, but wanted to flag

Reply:
wow umm good catch. i just totally brain farted there and read it as *un*batched, which is definitely what you want here